docs/anonymizer/index.md

```diff
@@ -244,6 +244,7 @@ of the AnalyzerEngine"
 | Anonymize     | mask          | Replace the PII with a given character                              | `chars_to_mask`: the amount of characters out of the PII that should be replaced. <br> `masking_char`: the character to be replaced with. <br> `from_end`: Whether to mask the PII from it's end. |
 | Anonymize     | encrypt       | Encrypt the PII using a given key                                   | `key`: a cryptographic key used for the encryption.                                                                                                                                               |
 | Anonymize     | custom        | Replace the PII with the result of the function executed on the PII | `lambda`: lambda to execute on the PII data. The lambda return type must be a string.                                                                                                             |
+| Anonymize     | surrogate     | Generate realistic, medically-appropriate surrogates using Azure Health Data Services SurrogateOnly operation | `endpoint`: AHDS endpoint (optional, uses AHDS_ENDPOINT env var)<br>`entities`: List of entities detected by analyzer<br>`input_locale`: Input locale (default: "en-US")<br>`surrogate_locale`: Surrogate locale (default: "en-US")<br>Requires: `pip install presidio-anonymizer[ahds]` |
```

Comment:
Can we consider renaming the operator to `surrogate_ahds`? This would allow us to create a fully OSS surrogate operator in the future, and not break backward compatibility.